src/res_printer.ml

```diff
@@ -3658,6 +3658,7 @@ and printBinaryExpression ~customLayout (expr : Parsetree.expression) cmtTbl =
     Doc.group
       (Doc.concat
          [
+           printAttributes ~customLayout expr.pexp_attributes cmtTbl;
```

Comment:
seen this one before perhaps?
https://github.com/rescript-lang/syntax/pull/621/files#diff-5f509cef22744d6c22d0c7a7fc37bcb68a1bda23db28caa9dc2b20cdbfebb695R3661